product_owners.md

```diff
@@ -0,0 +1,37 @@
+# Product Owners
+
+We strive for each of our applications to have a product owner (PO). The product
+owner is usually a key stakeholder for the application who can collaborate
+closely with us to guide development. Documentation of our team's [work cycles](/work_cycles.md) and [technical liaison role](/technical_liaisons.md) should give product owners useful context for this collaboration.
+
+The exact practices of each product owner will vary according to the application and the
+individuals themselves. Generally, product owners will work to:
+
+* Maintain a list of the highest priority issues for stakeholders on an ongoing
+basis.
+  * We recommend keeping track of these either in Github using a list of 10
+  "On Deck" labeled issues or using Zenhub to organize these to the top of the
+  project board's "Ready" column.
+  * The Technical Liaison for your project is available to support the PO in this
+  effort.
+* Participate during relevant work cycles by attending the planning meeting and
+  Friday wrap-up meeting when the PO's application is a focus of the cycle.
+* Be available on slack during a relevant work cycle to answer questions and
+  give feedback.
+* Bring in DLS developers when discussing new features or changes to the
```

Comment:
"bring in DLS developers when discussing new features or [potential] changes to..."